SimpleAdjustableFairings/ModelData.cs

```diff
@@ -50,7 +50,7 @@ public ResolvedModelData Resolve(GameObject lookupRoot)
 
             if (prefab == null) throw new ObjectNotFoundException(transformName);
 
-            return new ResolvedModelData(prefab, mass, CoM, rootOffset);
+            return new ResolvedModelData(prefab, mass * scale * scale, CoM * scale, rootOffset * scale);
```

Comment:
Actually hmm, why is multiplying the rootOffset by scale necessary?  Shouldn't it not matter since TweakScale adjusts the whole model?